cactusbot/cached/caching.py

```diff
@@ -0,0 +1,18 @@
+class CacheUtils(object):
+
+    # You have found MysticalMage's code, turn back now, no good lies ahead
+    def in_cache_followers(user):
+        """ Returns if user is in the cache. """
+        follower_cache = open("caches/CacheFollowers.txt", "r")
```

Comment:
We should use a better format than `.txt` for storing this. Using JSON would be much cleaner.